src/commands/environment_commands.c

```diff
@@ -1,13 +1,13 @@
 #include <commands/commands.h>
 #include <env/environment.h>
 #include <parser/parser.h>
+#include <stdio.h>
 
 t_exit_code	export_command(t_command command, t_output_stdout output)
 {
 	(void)output;
 	skip_spaces(&command.arg.start);
-	export(get_environment(), command.arg.start);
-	return (SUCCESS);
+	return ((t_exit_code) !export(get_environment(), command.arg.start));
```

Comment:
Why don't we change the prototype of export to be an exit code as well? Can be `SUCCESS` or `ERROR` right?